pulp_2_tests/constants.py

```diff
@@ -329,6 +329,15 @@
 This path may be joined with :data:`RPM_RICH_WEAK_FEED_URL`.
 """
 
+RPM_RICH_WEAK_DATA = MappingProxyType({
+    'name': 'Cobbler',
+    'rich_dependency': ['Scotch', 'contireau', 'icecubes', 'tablespoon-sugar'],
+    'weak_dependency': ['orange-bits'],
+    'total_dependencies': 5,
```

Comment:
Something I thought about to hopefully allow for more flexibility:

```python
RICH_DEPENDENCY = ['Scotch', 'Cointreau', 'Ice cubes', 'tablespoon-sugar']
WEAK_DEPENDENCY = ['orange-bits']

RPM_RICH_WEAK_DATA = MappingProxyType({
    'name': 'Cobbler',
    'rich_dependency': RICH_DEPENDENCY,
    'weak_dependency': WEAK_DEPENDENCY,
    'total_dependencies': len(RICH_DEPENDENCY),
    'total_installed_packages': len(RICH_DEPENDENCY) + len(WEAK_DEPENDENCY),
})
```

Also, should `total_dependencies == 4` and not `5`?

Reply:
`total_dependencies` is equal to total of weak + rich. 
`total_installed_packages` is equal to the `total_dependencies` + the package itself.